Source/Core/BoundingRectangle.js

```diff
@@ -318,6 +320,23 @@ define([
         return Intersect.OUTSIDE;
     };
 
+    /**
+     * @private
+     */
+    BoundingRectangle.closestLocationIn = function(position, boundingObject) {
```

Comment:
`BoundingRectangle` looks like it's in screen space, so I'm not sure this makes sense for a camera limiter